tests/test_data/scratch/README

```diff
@@ -0,0 +1 @@
+This directory exists for tests to temporarily write files. Anything in here except this readme can be safely deleted.
```

Comment:
I don't know if it really makes a difference, but we could use pytests `tmpdir`fixture for this instead? (https://docs.pytest.org/en/6.2.x/tmpdir.html)